src/ViewMarc.js

```diff
@@ -10,32 +10,52 @@ import {
 
 class ViewMarc extends React.Component {
   static manifest = Object.freeze({
+    query: {},
     marcRecord: {
       type: 'okapi',
       path: 'source-storage/formattedRecords/:{id}?identifier=INSTANCE',
-    }
+      accumulate: true,
+      throwErrors: false,
+    },
   });
 
+  state = { marcRecord: null };
+
+  componentDidMount() {
+    const {
+      marcRecord,
+      mutator: { marcRecord: connection },
+      onClose,
+    } = this.props;
+
+    if (!marcRecord) {
+      connection.GET()
+        .then(data => this.setState(state => ({ ...state, marcRecord: data })))
```

Comment:
```suggestion
        .then(data => this.setState({ marcRecord: data }))
```
It is not required to spread the state when updating it. React does it under the hood.
And you don't have to pass a function; object is enough.
https://reactjs.org/docs/state-and-lifecycle.html#state-updates-are-merged